File: api/src/main/kotlin/org/team619/rutor/model/BlankElement.java
package org.team619.rutor.model;

import org.jsoup.nodes.Attributes;
import org.jsoup.nodes.Element;
import org.jsoup.parser.Tag;

/**
 * Created by BORIS on 27.08.2016.
 */
public class BlankElement extends Element {

    public BlankElement() {
        super(Tag.valueOf("br"), "");
    }

    public BlankElement(Tag tag, String baseUri) {
        super(tag, baseUri);
    }

    public BlankElement(Tag tag, String baseUri, Attributes attributes) {
        super(tag, baseUri, attributes);
    }

    public static BlankElement newOne() {
        return new BlankElement();
    }

    @Override
    public void remove() {

    }

    @Override
    public String attr(String attributeKey) {
        return "";
    }

    @Override
    public String toString() {
        return "";
    }

}

File: api/src/main/kotlin/org/team619/rutor/model/BlankElement.kt
package org.team619.rutor.model

import org.jsoup.nodes.Element
import org.jsoup.parser.Tag

/**
 * Created by BORIS on 27.08.2016.
 */
class BlankElement : Element {

    constructor() : super(Tag.valueOf("br"), "")

    override fun remove() {

    }

    override fun attr(attributeKey: String): String {
        return ""
    }

    override fun toString(): String {
        return ""
    }

    companion object {

        fun newOne(): BlankElement {
            return BlankElement()
        }
    }

}
